docs/pages/eas-update/code-signing.mdx

```diff
@@ -4,52 +4,64 @@ description: Learn how code signing and key rotation work in EAS Update.
 ---
 
 import Head from '~/components/Head';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
 
 <Head title="EAS Update Code Signing" />
```

Comment:
Out of curiosity (and unrelated to this PR) what's the Head component's significance here?

Reply:
I believe that it is used to overwrite the page title in metadata, not sure if this still works as expected tho:
* https://github.com/expo/expo/blob/a41bfcff352d956d7284d20aa32130554e8634f6/docs/pages/eas-update/code-signing.mdx?plain=1#L10

Reply:
I think we can safely remove it. If required, let's alter the page title instead of using `Head` component (for consistency reasons).

Reply:
Sure, should we address this in this PR, or it's fine to change it in a separate one? Might be nice to also check if we use the `Head` link that on the other pages.

Reply:
Sure, addressing this in separate PR sounds good to me 👍

Reply:
Finally, I have done corrected this inside this fixes batch. 😄